frontend/apps/filemanager/filemanager.lua

```diff
@@ -1052,6 +1052,23 @@ function FileManager:moveFile(from, to)
     return BaseUtil.execute(self.mv_bin, from, to) == 0
 end
 
+--[[
```

Comment:
Here's a proper example:

https://github.com/koreader/koreader/blob/be72203b8302730b69f92b34acecb97baa96a83a/frontend/util.lua#L107-L111

Reply:
The suggestions I included are for the proper multiline style, but there is no reason here to use multiline comments.